docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
   - temporal cloud nexus
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
 :::tip SUPPORT, STABILITY, and DEPENDENCY INFO
 
-Temporal Nexus is available in [Public Preview](https://docs.temporal.io/evaluate/development-production-features/release-stages#public-preview) for both self-hosted deployments and Temporal Cloud with the [Temporal Go SDK](https://docs.temporal.io/develop/go).
+**Temporal Nexus** is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview) for both [Temporal Cloud](/cloud/nexus) and [self-hosted deployments](/production-deployment/self-hosted-guide/nexus).
+- [Temporal Go SDK](/develop/go/nexus) support for Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
+- [Temporal Java SDK](/develop/java/nexus) support for Nexus is available in [Pre-release](/evaluate/development-production-features/release-stages#pre-release).
 
 :::
 
-Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries.*
-It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
+Nexus allows you to reliably connect your Temporal Applications.
+It was designed with Durable Execution in mind and enables each team to have their own Namespace for improved modularity, security, troubleshooting, and fault isolation.
```

Comment:
Continuing from above thoughts, I think this could be stronger: Designed for Durable Execution, Nexus offers team-owned Namespaces that enhance modularity. Nexus establishes secure boundaries, separates ownership and responsibilities, enables precise troubleshooting, and improves fault isolation. <-- The points derive from modularity, so maybe they deserve their own sentence.

Reply:
thanks! and took parts of this and really like the stronger opening "Designed for Durable Execution, ..."